science/paraview/Portfile

```diff
@@ -97,39 +98,39 @@ configure.pre_args-delete \
 configure.post_args-append \
     -DCMAKE_BUILD_WITH_INSTALL_RPATH:BOOL=OFF
 
-# avoid `Exit code: 2`
-use_parallel_build      no
-
 # ${destroot} needs to be part of MACOSX_APP_INSTALL_PREFIX; see further
 # comments below, JJS 12/21/15
 configure.args-append \
     -DBUILD_TESTING:BOOL=OFF \
     -DMACOSX_APP_INSTALL_PREFIX=${destroot}${applications_dir} \
     -DBUILD_SHARED_LIBS:BOOL=ON \
     -DPARAVIEW_QT_VERSION=5 \
+    -DPARAVIEW_USE_MPI:BOOL=ON \
+    -DMPI_C_COMPILER=${mpi.cc} \
+    -DMPI_CXX_COMPILER=${mpi.cxx} \
+    -DMPIEXEC=${mpi.exec} \
+    -DMPI_Fortran_COMPILER=${mpi.fc} \
+    -DVTK_USE_SYSTEM_DIY2:BOOL=OFF \
     -DQt5_DIR=${qt_dir} \
     -DQT_HELP_GENERATOR=${qt_bins_dir}/qhelpgenerator \
     -DQT_XMLPATTERNS_EXECUTABLE=${qt_bins_dir}/xmlpatterns \
     -DVTK_USE_SYSTEM_LIBRARIES:BOOL=ON \
     -DVTK_USE_SYSTEM_XDMF2:BOOL=OFF
 
-pre-configure {
-    configure.args-append -DCMAKE_CXX_COMPILER=${configure.cxx} \
-        -DCMAKE_C_COMPILER=${configure.cc}
-
-       if {[mpi_variant_isset]} {
-           configure.args-append  \
-               -DPARAVIEW_USE_MPI:BOOL=ON \
-               -DMPI_C_COMPILER=${mpi.cc} \
-               -DMPI_CXX_COMPILER=${mpi.cxx} \
-               -DMPIEXEC=${mpi.exec} \
-               -DMPI_Fortran_COMPILER=${mpi.fc}
-       }
+
+if {[vercmp ${xcodeversion} 9.0] == 0} {
```

Comment:
Is the issue *really* only specific to Xcode 9.0, and not other 9.x versions ?
What about newer Xcode releases as well ?
Reading the github issue below it would appear to me you want to apply this to all 9.x and probably newer Xcodes.

Reply:
On newer Xcode versions, there's no problem. In the sense that for me, on Mojave, I don't need that fix. 

So what do you suggest to do? I check only if we are `>=9.0' and `< 10.0`?

Reply:
yes.